software_training_assignment/src/resetTurtle.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+Server for the Reset Turtle Service
+*/
+
+#include <ros/ros.h>
+#include <string> 
+
+//Including the service files for resetTurtle service
+#include <software_training_assignment/resetTurtle.h>
+
+//Struct for the service information 
+struct turtleInfo {
+    std::string name; 
+    int x_coord; 
+    int y_coord; 
+}
+
+//Defining the service class 
+class TurtleReset {
+public:
+    TurtleReset() {}
+
+    bool resetTurtlePosition(software_training_assignment::resetTurtleRequest &request, 
+                            software_training_assignment::resetTurtleResponse &response){
+        ROS_INFO("Turtle Position Reset"); 
```

Comment:
this function doesnt look like it does anything, shouldnt it reset turtle position